packages/riipen-ui/src/components/Drawer.test.jsx

```diff
@@ -0,0 +1,177 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Drawer from "./Drawer.jsx";
+
+describe("<Drawer>", () => {
+  it("renders without errors", () => {
+    let error;
+    try {
+      mount(<Drawer />);
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("snapshot test when open is true and onClose is provided", () => {
+    const open = true;
+    const onClose = jest.fn();
+
+    const wrapper = mount(<Drawer open={open} onClose={onClose} />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const defaultProps = new Drawer().type.defaultProps;
+
+      const wrapper = mount(<Drawer />);
+
+      const component = wrapper.find("Drawer");
+      expect(component.props().anchor).toEqual(defaultProps.anchor);
+      expect(component.props().open).toEqual(defaultProps.open);
+
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+
+    it("appends higher order values to default classes prop with withClass decorator", () => {
+      const classes = ["riipen", "riipen-drawer"];
+
+      const wrapper = mount(<Drawer />);
+
+      expect(
+        wrapper
+          .find("Drawer")
+          .props()
+          .classes.sort()
+      ).toEqual(classes.sort());
+    });
```

Comment:
I'm not this this test should be nested within the `default props` describe blocks. We aren't really testing the default props here, we are testing something else.